packages/blocks/src/components/import-page/styles.ts

```diff
@@ -55,10 +55,11 @@ export const styles = css`
   .button-container icon-button {
     padding: 8px 12px;
     justify-content: flex-start;
-    gap: 8px;
+    gap: 12px;
     width: 190px;
     height: 40px;
     box-shadow: var(--affine-shadow-1);
+    border-radius:10px;
```

Comment:
This fails the linter CI job again. The suggestion is to use VSCode with the [recommended extensions](https://github.com/toeverything/blocksuite/blob/master/.vscode/extensions.json) installed, so that you get proper lint warnings.

Reply:
Thank you